### DISCUSSION TITLE
502 Bad Gateway Error on almost every action. When creating a classroom, updating a roster,  or creating an assignment

### DISCUSSION THREAD

**Discussion by grentank** [community/community#114575] [GitHub Education]
### Hi everyone,

Third day in a row I experience 502 Bad Gateway Errors in github classroom.
I've been using Github Classroom for a couple of months and had no issues whatsoever.

Now I get the 502 Bad Gateway Error when:
- creating a classroom,
- updating a roster, 
- creating an assignment

I use Web version
Chrome browser 123.0.6312.86
No VPNs

**Comment by run3unblockedaz** [comment]
I am blown away by how you have compiled all of this knowledge for our benefit read Quran <a href="https://run3donline.com/"> **run 3**</a> of red color/a> Read on the internet.

**Comment by daniel-hasan** [comment]
I had the same problem, and I've fixed it by clearing my browser cache.

**Comment by lyannabueno** [comment]
Response #2

**Comment by samus-aran** [comment]
Hi @grentank 👋 

Are you still experiencing this?

**Reply by grentank** [reply]
I've fixed the issue by deleting all cookies with a domain `classroom.github.com` . Maybe clearing the browser cache could help too
